tooling/bundler/src/bundle/windows/nsis.rs

```diff
@@ -119,10 +118,12 @@ fn get_and_extract_nsis(nsis_toolset_path: &Path, _tauri_tools_path: &Path) -> c
     NSIS_TAURI_UTILS_SHA1,
     HashAlgorithm::Sha1,
   )?;
-
-  let target_folder = nsis_plugins.join("x86-unicode");
-  create_dir_all(&target_folder)?;
-  write(target_folder.join("nsis_tauri_utils.dll"), data)?;
```

Comment:
`x86-unicode` could be empty when building on Linux or macOS so we need to add this back

Reply:
Why does this code run on Linux and MacOS?

Reply:
because we support creating NSIS installers on macOS and Linux